CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change log
 
+## 0.8.0
+May 26th, 2023
+
+### Upgrade Notes
+* The support for different record creation queues is not a breaking change. Existing states are still supported.
+* All changes are from [#116](https://github.com/jwplayer/southpaw/pull/116)
+
+### Bug Fixes
+* Fixes an issue where backups, commits, and metrics reporting would not happen when a very large amount of denormalized records to be created were queued and created  
+
+### New Features
+* Changes how new primary keys for denormalized records are queued. There are now 3 different priority queues: low, medium, high. By default, everything is queued as medium, but a custom QueueingStrategy class can be specified using the `queueing.strategy.class` config option. Additionally, there is a "none" priority that causes the keys to be dropped.
```

Comment:
`queueing` should be `queuing` I believe

Reply:
Both are correct and `queueing` is used in the code.

Reply:
Oh are they? I learned something today